.github/workflows/test_dbt_models.yaml

```diff
@@ -22,25 +34,49 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
 
+      - name: Restore dbt state cache
+        id: cache
+        uses: ./.github/actions/restore_dbt_cache
+        with:
+          path: ${{ env.PROJECT_DIR }}/${{ env.STATE_DIR }}
+          key: ${{ env.CACHE_KEY }}
+
       - name: Install Python requirements
         run: pip install -r scripts/requirements.run_iasworld_data_tests.txt
         working-directory: ${{ env.PROJECT_DIR }}
         shell: bash
 
       - name: Run tests
         run: |
-          python3 scripts/run_iasworld_data_tests.py \
-            --target "$TARGET" \
-            --output-dir ./qc_test_results/
+          if [[ $CACHE_HIT == 'true' ]]; then
+            # shellcheck disable=SC2086
+            python3 scripts/run_iasworld_data_tests.py \
+              --target "$TARGET" \
+              --output-dir ./qc_test_results/ \
+              $SELECT_OPTION \
+              $SKIP_ARTIFACTS_OPTION \
+              --defer --state "$STATE_DIR"
+          else
+            # shellcheck disable=SC2086
+            python3 scripts/run_iasworld_data_tests.py \
+              --target "$TARGET" \
+              --output-dir ./qc_test_results/ \
+              $SELECT_OPTION \
+              $SKIP_ARTIFACTS_OPTION
+          fi
```

Comment:
It's annoying that these commands are identical, except that we add the `--defer` and `--state` flags in the case when `CACHE_HIT` is `true`. Is there a cleaner way to do this in bash that's still readable?

Reply:
Great idea, done!